travis/scripts/05-run.sh

```diff
@@ -47,9 +47,13 @@ launchCurlOrProtractor() {
         if [[ -f "gulpfile.js" ]]; then
             gulp itest --no-notification
         elif [[ -f "tsconfig.json" ]]; then
-            wget https://chromedriver.storage.googleapis.com/2.37/chromedriver_linux64.zip
-            unzip chromedriver_linux64.zip
-            yarn e2e --chromeDriver=./chromedriver
+            # Actually, `yarn e2e` could cause problems due to Github Search API rate limit.
+            # As we don't use gecko, we could use workaround below.
+            # See also https://github.com/angular/webdriver-manager/issues/216
+            set -ex
+            webdriver-manager update --gecko false
+            find -name "chromedriver*" ! -name "*.zip" -exec yarn e2e --chromeDriver={} \;
```

Comment:
@pascalgrimaud 
Thanks ;-).
But it's not very optimized.
This previous lines are retrieved each time `05-run.sh` is executed (four times). The best is to put this in [.travis.yml](https://github.com/jhipster/generator-jhipster/blob/312afc10c7ed5020886b96c23e4e7677104259d6/.travis.yml#L70), but I don't know if you like this. 
At [l.70 of this file](https://github.com/jhipster/generator-jhipster/blob/312afc10c7ed5020886b96c23e4e7677104259d6/.travis.yml#L70) we could add something like: 

```yaml
    - cd $APP_FOLDER
    - yarn add --dev webdriver-manager
    - ./node_modules/.bin/webdriver-manager update
    - export CHROMEDRIVER_PATH=`find -name "chromedriver*" ! -name "*.zip"`
```
then simplify `05-run.sh` simply by
```bash
yarn e2e --chromeDriver=$CHROMEDRIVER_PATH
```

But I don't know if you like something like that ?